res/themes/legacy-common/css/_legacy-custom.scss

```diff
@@ -17,28 +17,21 @@ limitations under the License.
 $font-family: var(--font-family, $font-family);
 $monospace-font-family: var(--font-family-monospace, $monospace-font-family);
 
-// Colors from Figma Compound https://www.figma.com/file/X4XTH9iS2KGJ2wFKDqkyed/Compound?node-id=559%3A741
 $accent: var(--accent, $accent);
 $alert: var(--alert, $alert);
 $links: var(--links, $links);
-$primary-content: var(--primary-content, $primary-content);
-$secondary-content: var(--secondary-content, $secondary-content);
-$tertiary-content: var(--tertiary-content, $tertiary-content);
```

Comment:
`primary-content` is redefined later on in the same file: https://github.com/matrix-org/matrix-react-sdk/pull/7726/files/21b8d90ca9cc4eb0e2b9332ccf70575d88fdcb19#diff-b489440c03bfaa8d465751827c8382ec7567b730e4362f9166d4c205942f3d49R68

Reply:
`secondary-content` would be redefined by `secondary-fg-color` by both the light and dark theme:
https://github.com/matrix-org/matrix-react-sdk/pull/7726/files/21b8d90ca9cc4eb0e2b9332ccf70575d88fdcb19#diff-ddc2c6eb3509b80549b30df3977708adfc119bfcea6e9253f88133791998666bL94
https://github.com/matrix-org/matrix-react-sdk/pull/7726/files/21b8d90ca9cc4eb0e2b9332ccf70575d88fdcb19#diff-90861a924b357562176a919e52e7efbe2f21df686d105dd439fcf244c9d51108L147

Thus no matter what the user defined for `secondary-content` it would not be respected in the theme. Since `secondary-content = $secondary-fg-color` is common for both light and dark themes it was moved into the newly created `legacy_common` file, which holds higher level css variables.